arkouda/sorting.py

```diff
@@ -7,13 +9,13 @@
 
 __all__ = ["argsort", "coargsort", "sort"]
 
-def argsort(pda : Union[pdarray,Strings]) -> pdarray:
+def argsort(pda : Union[pdarray,Strings,'Categorical']) -> pdarray:
```

Comment:
@hokiegeek2 did you need a `Categorical = ForwardRef('Categorical')` in this file

Reply:
@mhmerrill No, I tried to do that but circular dependencies are preventing putting in the actual Categorical classname, so no, not needed in this file.

Reply:
ok